book/src/leader-rotation.md

```diff
@@ -27,12 +102,25 @@ Leader schedule is generated using a predefined seed.  The process is as follows
    stake-weighted ordering.
 5. This ordering becomes valid after a cluster-configured number of ticks.
 
+## Schedule Attack Vectors
+
+### Seed
+
 The seed that is selected is predictable but unbiasable.  There is no grinding
-attack to influence its outcome. The active set, however, can be biased by a
-leader by censoring validator votes. To reduce the likelihood of censorship,
-the active set is sampled many slots in advance, such that votes will have been
-collected by multiple leaders. If even one node is honest, the malicious
-leaders will not be able to use censorship to influence the leader schedule.
+attack to influence its outcome. 
+
+### Active Set
+
+The active set, however, can be biased by a leader by censoring validator votes.
+To reduce the likelihood of censorship, the active set is sampled many slots in
+advance, such that votes will have been collected by multiple leaders. If even
```

Comment:
It'd be nice to make this section more precise. The simplest thing to do is calculate the active set right at the leader schedule offset. Is that the right place?